Vagrantfile

```diff
@@ -73,4 +80,20 @@ Vagrant.configure("2") do |config|
       type: "shell",
       inline: BOOTSTRAP
   end
+
+  (1..9).each do |i|
```

Comment:
Not sure this is the right approach. Once we support multi-node clusters, I'd consider to let Vagrant set up some standard infra, and indeed maybe allow to add some more, though have multi-node by default, with node-names that reference their role.

Also, the Vagrant provisioning should then set up the whole cluster.

Reply:
About the provisioning, I think we should do everything from the bootstrap node (except starting the new VMs of course). So other nodes won't probably ever have a provisioning step.

Yeah I'm not sure about the approach we want to have. One thing that I think is already settled is that we can not have a too-big cluster otherwise we won't be able to launch it on any laptop. Hence having not autostarted VMs.
But yeah we could have dedicated nodes to some roles. Then we could easily define all this in some yaml file and import it and use it into the Vagrantfile if we need to. We would just have to change the loop.

Since we don't really know what we want to test, I'd say it could be good enough to have access to those extra VMs in the meantime.

Reply:
From a user perspective:
- having a dedicated bootstrap node running for the E2E bootstrap test is a necessity
- having other nodes started is not - we don't need multiple nodes to confirm high-level bootstrap functions. 

To me the multi-VM thing would only be important for multi-node E2E tests, which we don't necessarily want to launch every time. Maybe we want a dedicated VagrantFile for these?

Although we probably don't need that many nodes to be ready, I'm happy with this. We can revise it once the need for multiple purposes and corresponding VM definitions comes up.

Reply:
OK, then let's wait until we have a better understanding of which environments we want to build.